hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCommitArchiveLog.java

```diff
@@ -173,6 +189,49 @@ private boolean deleteArchivedInstants(List<HoodieInstant> archivedInstants) {
         throw new HoodieIOException("Failed to delete archived instant " + archivedInstant, e);
       }
     }
+
+    // Remove older meta-data from auxiliary path too
+    Optional<HoodieInstant> youngestCommitted =
```

Comment:
Rename of oldestCommitted or latestCommitted based on what you mean by youngest..

Reply:
+1  left a similar comment above

Reply:
Done